content/knowledge-base/accessing-git-lfs.md

```diff
@@ -0,0 +1,12 @@
+---
+description: How to use git lfs with codemagic
+title: Accessing Git LFS
+weight: 9
+aliases:
+  - '../custom-scripts/accessing-git-lfs'
+---
+
+
+Git Large File Storage (LFS) replaces large files such as audio samples, videos, datasets, and graphics with text pointers inside Git, while storing the file contents on a remote server like GitHub.com or GitHub Enterprise. More info about Git LFS can be found [here]("https://git-lfs.github.com/").
+
+In order to access Git LFS (an open-source Git extension for versioning large files) during a build, add `git lfs install --skip-smudge` in a post-clone script.
```

Comment:
maybe a couple of words where the script needs to be added in flutter workflow editor vs yaml?